PlainSql.Migrations.Tool/PlainSql.Migrations.Tool.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <PackageOutputPath>./nupkg</PackageOutputPath>
+    <ToolCommandName>migrate</ToolCommandName>
+    <Description>A .NET Global tool that executes PlainSql.Migrations</Description>
+    <PackageTags>sql;migrations;conventional;commit;git</PackageTags>
+    <Company>Softwarepark</Company>
+    <Authors>Jonathan Channon</Authors>
```

Comment:
I guess a valid nuget user should be used here

Reply:
nuget doesn't actually link back to the user, it only links to the owner of the publisher. If you look here https://www.nuget.org/packages/PlainSql.Migrations/ for example

Reply:
@jchannon pro tip: never ask me to coin a catchy name 🥇